packages/cashier/src/containers/routes/binary-routes.tsx

```diff
@@ -19,21 +19,38 @@ const Loading = () => {
     );
 };
 
+const cashierV2RoutesConfig = {
+    p2p_v2: {
+        path: routes.cashier_p2p_v2,
+        getTitle: () => localize('Deriv P2P-V2'),
+        icon_component: 'IcDp2p',
+    },
+    cashier_v2: {
+        path: routes.cashier_v2,
+        getTitle: () => localize('Deriv Cashier-V2'),
+        icon_component: 'IcCashier',
+    },
+} as const;
+
 const BinaryRoutes = (props: TBinaryRoutesProps) => {
-    const { is_p2p_v2_enabled } = useFeatureFlags();
+    const { is_p2p_v2_enabled, is_next_cashier_enabled } = useFeatureFlags();
     const [routesConfig, setRoutesConfig] = React.useState(getRoutesConfig());
+
     useEffect(() => {
-        const is_p2p_v2_added = routesConfig[0].routes?.some(route => route.path === routes.cashier_p2p_v2);
-        if (is_p2p_v2_enabled && !is_p2p_v2_added) {
+        const isRouteAdded = (routePath: string) => routesConfig[0].routes?.some(route => route.path === routePath);
+
+        const should_add_p2p_v2_route = is_p2p_v2_enabled && !isRouteAdded(routes.cashier_p2p_v2);
```

Comment:
optional/nitpick/style: at some point we had discussion and as per updated /docs/JavaScript/README.md, the general preference is camelCase. snake_case is depreciated for new code (but given its partially new partially old, its kind of optional to update)

Reply:
since this package is legacy cashier, i will refactor it to snake_case @wojciech-deriv

Reply:
yes @wojciech-deriv since the changes is in legacy cashier package i used snake_case styling